### DISCUSSION TITLE
Removal of South Africa as region

### DISCUSSION THREAD

**Discussion by KarelSchwab** [supabase/supabase#3515] [Questions] [answered]
Hi everyone. I recently wanted to create a new Supabase project and realised that South Africa was removed as a region option. Is there a reason for the change? Is it a permanent change? Did I miss an announcement about it somewhere?

Thank you to the whole community for making such an awesome platform

**Comment by soedirgo** [comment, marked as answer]
Hey Karel, this is mostly caused by lack of feature parity on the AWS region: https://github.com/supabase/supabase/discussions/2949. Sorry for the troubles!

**Reply by KarelSchwab** [reply]
Thank you
